### PR TITLE
 [sipify] handles SIP_OUT arguments 

### PR DESCRIPTION
**Pull request by 3nids**
remove them from parameters list and add them to return values

### CODE REVIEW

File: python/core/auto_generated/qgsunittypes.sip.in [hunk 1]
```diff
@@ -152,9 +152,9 @@ Encodes a distance unit to a string.
 Decodes a distance unit from a string.
 
 :param string: string to decode
-:param ok: optional boolean, will be set to true if string was converted successfully
 
 :return: decoded units
+         ok:  optional boolean, will be set to true if string was converted successfully
```
Comment (nyalldawson) [inline python/core/auto_generated/qgsunittypes.sip.in]: Or maybe `(?:optional|if specified)[,]?\s*`

File: python/core/auto_generated/qgsvectorlayertools.sip.in [hunk 2]
```diff
@@ -86,9 +86,9 @@ Copy and move features with defined translation.
 :param request: The request for the features to be moved. It will be assigned to a new feature request with the newly copied features.
 :param dx: The translation on x
 :param dy: The translation on y
-:param errorMsg: If given, it will contain the error message
 
 :return: True if all features could be copied.
+         errorMsg:  If given, it will contain the error message
```
Comment (nyalldawson) [inline python/core/auto_generated/qgsvectorlayertools.sip.in]: (?:optional|if specified|if given)[,]?\s*

**Review by nyalldawson — COMMENTED** [review]
What's your plans for methods without an explicit \return ? SHould we hardcode in a new code layout rule that SIP_OUT methods MUST have a \returns in the doxygen?

**Comment by 3nids** [comment]
I will merge as is to see how it goes in sphinx.

TODOs:
* either a test to force \returns for methods having SIP_OUT or SIP_PYARGREMOVE
* regex to remove optional/if specified.

**Pull request merged**